### PR TITLE
Update the k8s versions

### PR DESCRIPTION
**Pull request by lwr20**
## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
TBD
```

## Reminder for the reviewer

Make sure that this PR has the correct labels and milestone set.

Every PR needs one `docs-*` label.

- `docs-pr-required`: This change requires a change to the documentation that has not been completed yet.
- `docs-completed`: This change has all necessary documentation completed.
- `docs-not-required`: This change has no user-facing impact and requires no docs.

Every PR needs one `release-note-*` label.

- `release-note-required`: This PR has user-facing changes. Most PRs should have this label.
- `release-note-not-required`: This PR has no user-facing changes.

Other optional labels:

- `cherry-pick-candidate`: This PR should be cherry-picked to an earlier release. For bug fixes only.
- `needs-operator-pr`: This PR is related to install and requires a corresponding change to the operator.

### CODE REVIEW

**Comment by lwr20** [comment]
This will need to be cherry picked to `release-v3.24`

File: calico/getting-started/kubernetes/requirements.md [hunk 1]
```diff
@@ -15,9 +15,10 @@ We test {{site.prodname}} {{page.version}} against the following Kubernetes vers
 - v1.22
```
Comment (caseydavenport) [inline calico/getting-started/kubernetes/requirements.md:15]: We should remove v1.22 as part of this PR. We keep the latest three releases.

Comment (lwr20) [inline calico/getting-started/kubernetes/requirements.md:15]:
```suggestion
```

**Comment by lwr20** [comment]
@caseydavenport would you mind re-reviewing please?

**Review by caseydavenport — APPROVED** [review]

**Pull request merged**